tests/fixtures/optimizer/normalize_identifiers.sql

```diff
@@ -1,21 +1,37 @@
 SELECT a FROM x;
 SELECT a FROM x;
 
+# dialect: snowflake
+SELECT A FROM X;
+SELECT A FROM X;
+
 SELECT "A" FROM "X";
 SELECT "A" FROM "X";
 
 SELECT a AS A FROM x;
 SELECT a AS A FROM x;
 
+# dialect: snowflake
+SELECT A AS a FROM X;
```

Comment:
why isn't the alias caps?

Reply:
For the same reason we don't lowercase them in the test case right above this

Reply:
it seems like it should though no?

Reply:
isn’t it an identifier?

Reply:
I think @barakalon wrote this pass, he might be able to provide more context. There is special handling for aliases:

```
# We need to leave the output aliases unchanged, so the selects need special handling
```

Reply:
ok fine